### ISSUE TITLE
Project jargon needs to be listed and defined for easy reference

### ISSUE THREAD

**Issue by strypey** [Openmedianetwork/OMN#4]
I notice that OMN discussions often involve hashtags, like #dotcon and #geekproblem, which are intended to convey specific concepts relevant to OMN projects. It would help if there was a list of these hastags, explaining exactly what they mean. This list could live on the wiki, or in a jargon file in this repo, or both. Any thoughts?

**Comment by mj-saunders** [comment]
agreed - wiki seems the more active location at the moment, so I guess in there.

**Comment by mj-saunders** [comment]
have added a page here: https://github.com/Openmedianetwork/OMN/wiki/Jargon

no definitions yet, and likely more tags to be added
